docs/src/components/usage.tsx

```diff
@@ -1,27 +1,84 @@
 import React from 'react'
 
-import { A } from './mdx/links'
+import Button from '@pluralsight/ps-design-system-button'
+import Icon, {
+  SocialGithubIcon,
+  ExternalLinkIcon,
+  ListIcon
+} from '@pluralsight/ps-design-system-icon'
 import * as styles from './usage.module.css'
 
 interface UsageProps {
+  figmaUrl: string
   install: string
   import: string
   packageName: string
   version: string
 }
 export const Usage: React.FC<UsageProps> = props => {
   return (
-    <ul className={styles.usage}>
-      <Item label="Version">
-        <A
-          href={`https://github.com/pluralsight/design-system/blob/master/packages/${props.packageName}/CHANGELOG.md`}
-        >
-          {props.version || 'CHANGELOG'}
-        </A>
-      </Item>
-      <Item label="Install">{props.install.trim()}</Item>
-      <Item label="Import">{props.import.trim()}</Item>
-    </ul>
+    <>
+      <ul className={styles.links}>
+        <li>
+          <Button
+            appearance="secondary"
+            href={`https://github.com/pluralsight/design-system/blob/master/packages/${props.packageName}`}
+            target="_blank"
```

Comment:
Please add companion attr `rel="noreferrer"` to these external links.

See https://web.dev/external-anchors-use-rel-noopener/

Reply:
👍